features/portfolio/pdf_extraction/__init__.py

```
@@ -0,0 +1,22 @@
+
+__all__ = [
+    "PdfActivity",
+    "PdfExtractionGenerationError",
+    "PdfExtractionGenerator",
+    "PdfExtractionResult",
+    "PdfProblemSolvingItem",
+    "PdfExtractionService",
+    "get_pdf_extraction_service",
+    "init_pdf_extraction_service",
+    "reset_pdf_extraction_service",
+]
```

Ruff (0.15.6)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)